### ISSUE TITLE
Cannot install `diff-gaussian-rasterization`

### ISSUE THREAD

**Issue by alanvinx** [humansensinglab/taming-3dgs#14]
Hi, when installing `diff-gaussian-rasterization` in my environment I got this error message:
```
"class at::Tensor __cdecl fusedssim(float,float,class at::Tensor &,class at::Tensor &)" (?fusedssim@@YA?AVTensor@at@@MMAEAV12@0@Z)
      build\lib.win-amd64-cpython-38\diff_gaussian_rasterization\_C.cp38-win_amd64.pyd : fatal error LNK1120
```

After modifying the output type of `fusedssim` here https://github.com/humansensinglab/taming-3dgs/blob/main/submodules/diff-gaussian-rasterization/rasterize_points.h#L112 from `std::tuple<torch::Tensor,torch::Tensor,torch::Tensor,torch::Tensor>` to `torch::Tensor` it worked.

**Comment by rahul-goel** [comment]
Hi.

Seems that the legacy ssim code wasn't removed correctly. Is it possible for you to confirm that the following branch works well: https://github.com/humansensinglab/taming-3dgs/tree/remove_old_ssim? It is working on mine. If it works for your case, I'll merge with main.

Thanks a lot.

**Comment by alanvinx** [comment]
Hello, I successfully installed `diff-gaussian-rasterization` after moving to this other branch, thanks!

**Comment by saswat0** [comment]
Fixed in #15
